store/sqlstore/supplier_test.go

```diff
@@ -142,6 +144,29 @@ func TestGetReplica(t *testing.T) {
 	}
 }
 
+func TestGetDbVersion(t *testing.T) {
+	driverName := model.DATABASE_DRIVER_SQLITE
```

Comment:
Why is this only tested for SQLITE?

Reply:
Basically, because all the tests on that file are using only SQLite, even the `GetReplica` one` so I assumed that we was just testing SQLite but I've checked that the other tested methods don't have a different query per database engine sooooo I'm going to add it, thx for pointing it out :)

Reply:
@jespino done I've made some refactor too